dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -602,6 +710,20 @@ const styles = {
 			justify-content: flex-end;
 		}
 	`,
+	threeTierChoiceCardsContainer: css`
+		order: 3;
+		${until.tablet} {
+			padding-bottom: 100px;
+		}
+		${until.desktop} {
+			padding-bottom: 120px;
```

Comment:
This approach ensures the CTA remains visible ("sticky") on tablet and mobile views.  It works by increasing the overall size of the choice cards container, preventing the CTA from covering the final choice card regardless of screen size or expanded choice cards.  
This issue is particularly noticeable at mobile breakpoints and on tablets with a header image.  
If anyone has a cleaner solution, feel free to suggest improvements!

Reply:
This will be an issue if we reduce the body copy down as we'll have a lot of empty space....it needs to be dynamic OR we have a confirmed word length from MRR before going live